src/storage/secondary/rowset/mem_rowset.rs

```diff
@@ -119,8 +120,9 @@ impl MemTable for ColumnMemTable {
 
 pub struct SecondaryMemRowset<M: MemTable> {
     mem_table: M,
-    rowset_builder: RowsetBuilder,
     rowset_id: u32,
+    directory: PathBuf,
```

Comment:
I think it's better to remove `directory` from memory rowset, and pass the directory as a parameter when doing flush.

Reply:
That's better! Thanks!